### DISCUSSION TITLE
Update Transitive Dependencies

### DISCUSSION THREAD

**Discussion by blitz** [renovatebot/renovate#15762] [Suggest an Idea]
### What would you like Renovate to be able to do?

Renovate currently updates direct dependencies of a project. In Rust with Cargo for dependencies, transitive dependencies are _not_ updated. This means that dependencies of dependencies still have to be manually updated via `cargo update` to avoid running into stale dependencies with security or other issues.

This relates to #13958.

### If you have any ideas on how this should be implemented, please tell us here.

Opening PRs for individual transitive dependencies will likely generate a ton of noise. As a user, I would want to have a weekly MR that just bumps all transitive dependencies.

### Is this a feature you are interested in implementing yourself?

No

**Comment by viceice** [comment]
what about lockfile maintenance feature? isn't that working for cargo?

**Comment by rarkins** [comment]
What is described here is exactly how lockFileMaintenance is intended to work

**Comment by blitz** [comment]
Ah, never mind then. It seems I just have issues finding this feature as the naming was unintuitive. Sorry for the noise!

**Reply by rarkins** [reply]
https://github.com/renovatebot/renovate/blob/main/docs/development/behavior.md#avoid-complaining-about-docs-after-getting-help
